data-model/fdo-types/0.3.0/digital-specimens/schema/identifications.json

```diff
@@ -0,0 +1,361 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/digitalobjects/0.3.0/digital-specimens/identifications.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment": "DigitalObject Version 0.3.0",
+  "type": "object",
+  "properties": {
+    "dwc:identificationID": {
```

Comment:
Changing to ID instead of Id is good, as it is basically implied as ID in most places